mirrord/layer/src/hooks.rs

```diff
@@ -101,12 +101,18 @@ impl<'a> HookManager<'a> {
     ) -> Result<NativePointer> {
         // This can't fail
         let module = self.modules.first().unwrap().clone();
+        // for symbol in Module::enumerate_symbols(&module) {
+        //     trace!("found symbol {}", symbol.name);
+        // }
+
         self.hook_symbol(&module, symbol, detour)
     }
 
     /// Resolve symbol in main module
-    #[cfg(target_os = "linux")]
-    #[cfg(target_arch = "x86_64")]
+    #[cfg(all(
+        target_os = "linux",
+        any(target_arch = "x86_64", target_arch = "aarch64")
```

Comment:
Do we need this arch selection (also in all other locations)?

Reply:
well, in the case someone tries to compile to mips or something we didn't implement.. doesn't hurt